### DISCUSSION TITLE
Could we have the time durations in hours, minutes and seconds?

### DISCUSSION THREAD

**Discussion by jackheli** [berrystephenw/OctoText#121] [General]
Would be great if we didn't need to covert the seconds duration information ourselves

**Comment by berrystephenw** [comment]
Are you using the latest version (0.3.3)? if you select this check box:
![image](https://user-images.githubusercontent.com/44931130/158245042-5162e9ce-2441-4a13-8ad2-3b2f59047bcc.png)
you will get H:M:S interval reporting.

**Reply by jackheli** [reply]
Excellent. Thank you for the super fast reply
